test/lint/lint-format-strings.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2018-2020 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+#
+
+"""
+Lint format strings: This program checks that the number of arguments passed
+to a variadic format string function matches the number of format specifiers
+in the format string.
+"""
```

Comment:
```suggestion
#!/usr/bin/env python3

"""
Lint format strings: This program checks that the number of arguments passed
to a variadic format string function matches the number of format specifiers
in the format string.

Copyright (c) 2018-2020 The Bitcoin Core developers
Distributed under the MIT software license, see the accompanying
file COPYING or http://www.opensource.org/licenses/mit-license.php.
"""
```

Reply:
Previously ported python scripts have the copyright comment in the way existing version of this PR has it. Changing it to the suggested way might look in-consistent with other lint files. Do you still suggest changing ?